services/datamanager/builtin/capture/capture_test.go

```diff
@@ -0,0 +1,32 @@
+package capture
```

Comment:
Is there anyway we can rename this file, its not clear to me how we split this between the other `capture_test.go`

Reply:
renamed